### PR TITLE
Add builders page, to list building related sub-pages

### PR DESCRIPTION
**Pull request by caldav**
* Add builders page with initial content
* Make build* pages sub-pages of builders

Misc: 

* surface Meson plugin in plugins list

### CODE REVIEW

File: build-snaps/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You can set up your repository to automatically build and publish snaps using [t
 ### Building from Ubuntu 14.04 or another Linux distribution
 
 If you're running the older Ubuntu 14.04 LTS or another Linux distribution, you can still build snaps using LXD or Docker. Just
-follow [these instructions](/docs/build-snaps/trusty).
```
Comment (evandandrea) [inline build-snaps/index.md]: Does the docs framework help us to not create a 404 here? There will still be pages linking to this which we cannot update.

File: build-snaps/builders.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Automating builds and releases
+
+When you have a working snap, the next step is to use [build.snapcraft.io](https://build.snapcraft.io) to link your code repository and automate `amd64` and `armhf` builds and releases.
+
+## Other builders
+
+There are many ways to build a snap:
+
+* [Build in containers with LXD or Docker](/docs/build-snaps/build-on-lxd-docker)
```
Comment (evandandrea) [inline build-snaps/builders.md]: Would you be amenable to splitting this into separate Docker and LXD pages?

**Comment by evandandrea** [comment]
One comment inline, otherwise I'm 👍

File: build-snaps/builders.md (hunk 2 above)
Comment (sergiusens) [inline build-snaps/builders.md]: Also to take into account, the former is driven by snapcraft while the latter by docker itself.

Comment (caldav) [inline build-snaps/builders.md]: @sergiusens not sure I understand, the LXD part of the page is for when snapcraft isn't available on the host system, so both cases are fairly similar. Set up container manually, run snapcraft inside it.

**Comment by caldav** [comment]
Merging this one for now, splitting later, as the redirect is ready to land as well

**Pull request merged**